packages/material-ui/src/styles/index.d.ts

```diff
@@ -7,7 +7,7 @@ export {
 } from './createPalette';
 export { default as createStyles } from './createStyles';
 export { TypographyStyle } from './createTypography';
-export { default as makeStyles } from './makeStyles';
+export * from './makeStyles';
```

Comment:
Point vscode directly to the file when doing `cmd+click`. Currently on going to definition it points to this LOC